src/app.py

```diff
@@ -121,3 +121,30 @@ async def submit_form(request: Request, selected: str = Form(...)):     # trunk-
                                               'title': "BTC Converter"})
     except Exception as e:
         logging.error(e)
+
+@app.get("/rate")
+async def get_rate(pair: str):
+    """
+    Retrieve the exchange rate between BTC and given currency.
+
+    Parameters:
+        - **param1** (str)
+        eg: btcusd
+
+    Returns:
+        dict: {"rate":"12,00.35"}
+    """
+    currency = pair[-3:]
```

Comment:
How about the inverse if the someone wants to know USD/BTC rate? ( as an example)

we've been working on adding sats conversions in addition to btc/fiat conversion as more people are using sats (100million sats = 1 btc) , also see the PR the charanravi is working on. (https://github.com/bitkarrot/satsconverter/pull/11)

I realize it was not mentioned in original issue but it would become more inline with the rest of project.

Everything else here looks acceptable. Thanks!

Reply:
Hi @bitkarrot .. 
I am re-iterating the requirement. Please correct me or add more points if I miss:

> Add Route to get exchange rate between BTC and any FAIT.
> The endpoint can be `/rate?pair=btc<anycurrency>` or `/rate?pair=<anycurrency>btc`

Do I need to work on anything related to sats? This is unclear from your comment.

Reply:
Hi @sudheersingampalli , thanks for the quick reply. The routes are correct. 

Question 2:
for the  `/rate?pair=hkdbtc gives 0.00000480 ? `conversion, this is correct.

Question 3:

btc =  0.00000480
sats = 480

where 100 Million Sats == 1 BTC. 

So the representation for small units is sats and is used more frequently these days for micro transactions. 

So an example of what you could return for each corresponding endpoint is: 

{"btchkd": "208,374.35"}

{"hkdbtc":"0.00000480"} 

{"hkdsat":  "480"}

{"sathkd": "......"}

I hope that makes sense.

Reply:
<img width="952" alt="Screenshot 2023-06-09 at 12 06 31 PM" src="https://github.com/bitkarrot/satsconverter/assets/73979971/48f4c974-f8f5-4324-af02-f0e20f28f676">

Reply:
Thanks for the explanation. Will work on it :)